src/pages/studyView/tabs/SummaryTab.tsx

```diff
@@ -325,6 +348,20 @@ export class StudySummaryTab extends React.Component<
                     props.promise = this.store.getGenericAssayChartDataCount(
                         chartMeta
                     );
+                } else if (
```

Comment:
when a chart type has subtypes (requiring these if/else statements), we should detect subtype here and then pass to helper function, which will use another switch statement or lookup table 
```
{
    [chartSubType]: (...)=>props
}

```